Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -152,6 +152,8 @@ typedef NS_ENUM(NSInteger, _WKImmediateActionType) {
 @class NSImage;
 #endif
 
+typedef struct _NSRange NSRange;
```

Comment:
Is this forward declaration necessary? I think we should already have pulled in all public Foundation headers here, since we have AppKit/UIKit.